ui/imports/utils/Constants.qml

```diff
@@ -15,20 +15,18 @@ QtObject {
         readonly property int wallet: 2
         readonly property int walletv2: 3
         readonly property int browser: 4
-        readonly property int timeline: 5
```

Comment:
As mentioned above, we should better have these decoupled. It doesn't make sense to lock `timeline` to `5` just because the chatType is `5`. Just like above, this is a value for `appSection` and we shouldn't have it depend on the value, nor even the existence of a chatType with the same value.

(there's also no chatType for browser, and wallet, so why having a special rule here?)

Reply:
@PascalPrecht I fully agree, actually in real they are not in corelation at all in the current code, this deletion is totally fine.